<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="btn_frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>150</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:none;
border-right:1px solid;
color:white;
font-size:20px;
font-family:&quot;Open-Sans&quot;,sans-serif;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="OkBtn">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#OkBtn{background: #fc7169;
}
#OkBtn:hover{
background-color: #fc8982;
}</string>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelBtn">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#CancelBtn{
background-color: #35a785;
}
#CancelBtn:hover{
	
	background-color: rgb(63, 199, 158);
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="msg_frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(253, 255, 239);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Err_msg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:#5e6e8d;</string>
    </property>
    <property name="text">
     <string>Registration Successful</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
